### ISSUE TITLE
Latest version - Use of unresolved identifier 'AppEventsLogger'

### ISSUE THREAD

**Issue by guidev** [facebook/facebook-swift-sdk#433]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `10.2.1`
- Swift Version: `5.0`
- Installation Platform & Verison: `[Cocoapods]` version `1.6.1`

## Goals

Upgrade from FacebookCore 0.6.0 to FacebookCore 0.7.0

## Expected Results

Being able to upgrade without any major change to the code. 

## Actual Results

AppEventsLogger is undefined.

## Code Samples & Details

My Podfile

```
platform :ios, '9.0'
use_frameworks!
inhibit_all_warnings!

target 'Cinema' do
  pod 'SDWebImage'
  pod 'Fabric'
  pod 'Crashlytics'
  pod 'Alamofire'
  pod 'ObjectMapper'
  pod 'FacebookCore'
  pod 'Firebase/Core'
  pod 'Firebase/Performance'
end

```

**Comment by maksymhusar** [comment]
The same for me

**Comment by Dfmarulanda** [comment]
Nothing is working in 0.7.0. 
I had to downgrade.

**Comment by ethanwa** [comment]
Same problem for me as well.

**Comment by iKisliy** [comment]
Same for me

**Comment by adrianorezena** [comment]
I've had the same problem and the changed the `AppEventsLogger.log` to `AppEvents.logEvent`.

It's compiling and logging for me now, but in the test events each log seems to be quadruplicated.
Does anyone had the same problem?

**Comment by iKisliy** [comment]
@adrianorezena did you had changed your .activate method? Neither AppEventsLogger.activate, nor AppEvents.activate works

**Comment by adrianorezena** [comment]
@iKisliy , have you tried AppEvents.activateApp()?

**Comment by AmrKafina** [comment]
Yup, spent more than two hours trying to integrate the SDK with my app.

**Comment by sglushchenko** [comment]
I added
`pod 'FacebookSDK'`
and use
```
import FBSDKCoreKit
...
AppEvents.activateApp()
```

**Comment by NikitaSkripchenko** [comment]
@Dfmarulanda on which one you had downgraded?

**Comment by mouness2020** [comment]
same problem here

**Comment by piyush-cashify** [comment]
Facing same issue

**Comment by geronimowheeli** [comment]
Having the same problem. Going to downgrade right now.

**Comment by Dfmarulanda** [comment]
13 days and @facebook-github-bot hasn't fixed this yet. SHAME

**Comment by idrougge** [comment]
One month…

**Comment by andrassomogyi** [comment]
Any workaround for this?

**Comment by kershner** [comment]
Can someone please walk through their steps for a working solution, via downgrading or otherwise?  This is still very much a problem, and I bet most people who go to implement the SDK in their Swift apps using the official FB docs are going to end up here sooner or later.

**Comment by piyush-cashify** [comment]
> Can someone please walk through their steps for a working solution, via downgrading or otherwise? This is still very much a problem, and I bet most people who go to implement the SDK in their Swift apps using the official FB docs are going to end up here sooner or later.

https://github.com/facebook/facebook-swift-sdk/issues/463#issuecomment-499367825

**Comment by rromanchuk** [comment]
The official docs are a complete mess, has anyone dug into the source to figure out what the actual signatures are to send analytics. All these language specific are bogus.

<img width="1052" alt="Screen Shot 2019-07-02 at 12 34 19 PM" src="https://user-images.githubusercontent.com/102188/60541192-df476800-9cc5-11e9-9200-cc8c423e1813.png">

**Comment by joesus** [comment]
So the long story is that this SDK is a wrapper around the [ObjC SDK](https://github.com/facebook/facebook-objc-sdk). There's a little more information about this in the recently merged [roadmap](https://github.com/facebook/facebook-swift-sdk/blob/master/ROADMAP.md).

TLDR: If you look at the changes between v0.6.0 and v0.7.0 - [compare](https://github.com/facebook/facebook-swift-sdk/compare/v0.6.0...v0.7.0) we updated to point to version 5.0 of the ObjC SDK. This was a breaking change and we should have cut a major release to reflect that. 

Those changes constituted a major effort to enhance the usability of the Swift interface that's generated from the ObjC SDK but it was not communicated well in terms of this project. The changes are documented reasonably well in this [changelog](https://github.com/facebook/facebook-objc-sdk/blob/master/CHANGELOG.md) under the 5.0 release.

While it is completely reasonable to be upset about this series of events, I hope that this is enough explanation to unblock people who upgraded. Closing the issue as this was a bug in an evolving process and cannot be fixed by a simple code change.

**Comment by moesalih** [comment]
@joesus the docs you're linking to don't say what changes were done to `AppEventsLogger`. What's the solution? This is not resolved.

**Comment by joesus** [comment]
@moesalih, good call. I just merged a pr to update the changelog so it's easier to see what was changed on the Swift SDK end of things. Check out: https://github.com/facebook/facebook-swift-sdk/compare/v0.6.0...v0.7.0

**Comment by ethanwa** [comment]
For those who have simple implementations, the answer to get things working is this change:

`AppEventsLogger.activate(application)`

is now

`AppEvents.activateApp()`
